src/helpers/tests/utils.test.ts

```diff
@@ -0,0 +1,14 @@
+import { timeout } from "../utils"
+
+describe("timeout", () => {
+  it("should return a promise after supplied ms have elapsed", async () => {
+    const TIMEOUT_MS = 1000
+    const MARGIN_OF_ERROR = 10
+
+    const t1 = new Date()
+    await timeout(TIMEOUT_MS)
+    const t2 = new Date()
+    const difference = t2.valueOf() - t1.valueOf()
+    expect(Math.abs(difference - TIMEOUT_MS)).toBeLessThan(MARGIN_OF_ERROR)
```

Comment:
I like that we're not doing an exact subtraction here. Otherwise, it'll result in a flaky test  because of time